package.json

```diff
@@ -11,7 +11,8 @@
     "build": "gulp build",
     "heroku-postbuild": "gulp build",
     "lint:prettier": "prettier --config --write '**/*.js'",
-    "test": "DEV=true mocha './test/**/*.js' --exit"
+    "test": "DEV=true mocha './test/**/*.js' --exit",
+    "dev-backend": "UPDATE_DATA=true DEV=true ts-node backend/app.ts"
```

Comment:
really small - but in our JS repos, we generally namespace dev environment scripts as `start`. Possible to rename this script to `start:backend`?

Reply:
good point, `start:backend` sgtm 👍 , changed
https://github.com/stellar/dashboard/pull/202/commits/f2c1471c5b79981dfa6cb9a8558531e60d3727ab